sdk/azidentity/chained_token_credential.go

```diff
@@ -36,28 +36,35 @@ func NewChainedTokenCredential(sources ...azcore.TokenCredential) (*ChainedToken
 // GetToken sequentially calls TokenCredential.GetToken on all the specified sources, returning the token from the first successful call to GetToken().
 func (c *ChainedTokenCredential) GetToken(ctx context.Context, opts azcore.TokenRequestOptions) (token *azcore.AccessToken, err error) {
 	var errList []*CredentialUnavailableError
-	for _, cred := range c.sources { // loop through all of the credentials provided in sources
-		token, err = cred.GetToken(ctx, opts) // make a GetToken request for the current credential in the loop
+	// loop through all of the credentials provided in sources
+	for _, cred := range c.sources {
+		// make a GetToken request for the current credential in the loop
+		token, err = cred.GetToken(ctx, opts)
+		// check if we received a CredentialUnavailableError
 		var credErr *CredentialUnavailableError
-		if errors.As(err, &credErr) { // check if we received a CredentialUnavailableError
-			errList = append(errList, credErr) // if we did receive a CredentialUnavailableError then we append it to our error slice and continue looping for a good credential
-		} else if err != nil { // if we receive some other type of error then we must stop looping and process the error accordingly
+		if errors.As(err, &credErr) {
+			// if we did receive a CredentialUnavailableError then we append it to our error slice and continue looping for a good credential
+			errList = append(errList, credErr)
+		} else if err != nil {
+			// if we receive some other type of error then we must stop looping and process the error accordingly
 			var authenticationFailed *AuthenticationFailedError
-			if errors.As(err, &authenticationFailed) { // if the error is an AuthenticationFailedError we return the error related to the invalid credential and append all of the other error messages received prior to this point
+			if errors.As(err, &authenticationFailed) {
+				// if the error is an AuthenticationFailedError we return the error related to the invalid credential and append all of the other error messages received prior to this point
 				authErr := &AuthenticationFailedError{msg: "Received an AuthenticationFailedError, there is an invalid credential in the chain. " + createChainedErrorMessage(errList), inner: err}
-				addGetTokenFailureLogs("Chained Token Credential", authErr)
 				return nil, authErr
 			}
-			addGetTokenFailureLogs("Chained Token Credential", err)
```

Comment:
Why did this get removed?

Reply:
The failing call(s) to each `GetToken()` in the chain already logged this error, so this added a duplicate log entry.

Reply:
Ah true. Ok sounds good then